File: src/codes/Users_Manual/segment_change.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = new ControllerBuilder().AddDevice(new AUTD3(Vector3d.Zero)).Open(Nop.Builder());
autd.Send(SwapSegment.Gain(Segment.S1));

File: src/codes/Users_Manual/segment_change.py
~from pyautd3 import Segment, SwapSegment
~from pyautd3 import Controller, AUTD3
~from pyautd3.link.audit import Audit
~autd: Controller[Audit] = Controller.builder().add_device(AUTD3([0.0, 0.0, 0.0])).open(Audit.builder())
autd.send(SwapSegment.gain(Segment.S1))
